Cargo.toml

```diff
@@ -16,7 +16,7 @@ colorgrad = "*"
 crossterm = { version = '0.23', features = ["event-stream"] }
 futures = "0.3"
 futures-timer = "3.0"
-image = "*"
+image = "0.24"  # TODO: tui-image (see below) doesn't allow newer versions.
```

Comment:
the general problem here is the dependency to the unmaintained tui-rs. I wrote a ticket to takle it 'some day'. https://github.com/carzum/termviz/issues/94